db/migrate/20160729182517_remove_system_add_source_to_miq_ae_namespace.rb

```diff
@@ -2,9 +2,11 @@ class RemoveSystemAddSourceToMiqAeNamespace < ActiveRecord::Migration[5.0]
   class MiqAeNamespace < ActiveRecord::Base; end
 
   def up
-    say_with_time('Migrating System attribute to Source in MiqAeNamespace') do
+    say_with_time('Adding source to MiqAeNamespace') do
```

Comment:
@mkanoor There is no need to say_with_time around built-in migration methods like `add_column`...they already do it under the covers.